src/base_classes/reference.h

```diff
@@ -168,7 +152,9 @@ class Reference // : public psi::Wavefunction
     ambit::Tensor g3abb_;
     ambit::Tensor g3bbb_;
 };
-} // namespace forte
 
+double compute_Eref_from_reference(Reference& ref, std::shared_ptr<ForteIntegrals> ints,
```

Comment:
Perhaps `const Reference& ref`.

Reply:
Probably a good idea, I'll make the changes now